arm-sql/2014-04-01/examples/ElasticPoolCreateMax.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-2369",
+    "serverName": "sqlcrudtest-8069",
+    "elasticPoolName": "sqlcrudtest-8102",
+    "api-version": "2014-04-01",
+    "parameters": {
+      "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-2369/providers/Microsoft.Sql/servers/sqlcrudtest-8069/elasticPools/sqlcrudtest-8102",
```

Comment:
from creates remove ID, name, location (remove for child resource only), kind, type for resources. We should not encourage users to send these values in. I have not cross checked if in swagger these are marked as readonly or not. These should be marked readonly, so that we don't generate setter for the same and don't send these across network